lib/suspenders/app_builder.rb

```diff
@@ -66,7 +66,7 @@ def configure_smtp
       copy_file 'smtp.rb', 'config/smtp.rb'
 
       prepend_file 'config/environments/production.rb',
-        %{require Rails.root.join("config/smtp")\n}
+        %{require Rails.root.join("config/smtp") if ENV["SMTP_ADDRESS"].present?\n}
```

Comment:
Line is too long. [83/80]<br>Align the parameters of a method call if they span more than one line.